File: BasketApplication/app/src/main/java/winners/mobile/application/GameActivity.java
package winners.mobile.application;

import android.content.DialogInterface;
import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.ImageView;

import androidx.appcompat.app.AlertDialog;
import androidx.appcompat.app.AppCompatActivity;

import winners.mobile.application.catch_the_ball.GameActivity2;

public class GameActivity extends AppCompatActivity {

    ImageView more;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_game);
    }

    public void startGame(View view){
        Intent intent=new Intent(getApplicationContext(), GameActivity2.class);
        intent.setFlags(Intent.FLAG_ACTIVITY_CLEAR_TASK|Intent.FLAG_ACTIVITY_NEW_TASK);
        startActivity(intent);
    }

    @Override
    public void onBackPressed()
    {
        AlertDialog.Builder builder = new AlertDialog.Builder(this);
        builder.setTitle("Exit");
        builder.setMessage("Are You Sure?");

        builder.setPositiveButton("Yes", new DialogInterface.OnClickListener() {
            public void onClick(DialogInterface dialog, int which) {
                dialog.dismiss();
                finish();
                System.exit(0);
            }
        });

        builder.setNegativeButton("No", new DialogInterface.OnClickListener() {
            @Override
            public void onClick(DialogInterface dialog, int which) {
                dialog.dismiss();
            }
        });
        AlertDialog alert = builder.create();
        alert.show();
    }
}

File: FottballGameApplication/app/src/main/java/pixel/game/GameActivity.kt
package pixel.game

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import com.example.testcaseapplication.databinding.GameActivityBinding

class GameActivity : AppCompatActivity() {
    lateinit var footballGameView: GameFootballView
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        val gameActivityBinding = GameActivityBinding.inflate(layoutInflater)
        footballGameView =  gameActivityBinding.footballGame
        footballGameView.textViewCounter = gameActivityBinding.counter
        setContentView(gameActivityBinding.root)
        gameActivityBinding.introductionButton.setOnClickListener{
            gameActivityBinding.introduction.visibility = if(gameActivityBinding.introduction.visibility== View.GONE) View.VISIBLE else View.GONE
        }
    }
    override fun onBackPressed() {
        setResult(RESULT_OK, Intent())
        finish()
    }
}